CraftTweaker2-API/src/main/java/crafttweaker/api/logger/MTLogger.java

```diff
@@ -61,14 +57,12 @@ public void logWarning(String message) {
             logger.logWarning(message);
         }
         
-        String message2 = "WARNING: " + message;
+        String message2 = "\u00a7eWARNING: " + message;
```

Comment:
That unicode was for coloring the message iirc.
Any reason to remove it?

Reply:
Coloring warnings and errors message can make them catch the eye. If you want, I can remove `§`.

Reply:
Oh, wait, I read that the other way round lmao